docs/book/src/plugins/available-plugins.md

```diff
@@ -0,0 +1,19 @@
+# Available plugins
+
+This section describes the plugins supported and shipped in with the Kubebuilder project.
+
+But any user can create their own plugins :
+- [Creating your own plugins](creating-plugins.md). 
+- [Plugins Phase 2](https://github.com/kubernetes-sigs/kubebuilder/issues/1378) - will allow users to use these external options as helpers to perform the scaffolds with the tool.
```

Comment:
WDYT about it be a note? Would not render better?

Reply:
I would also like to point to point to to the phase 2 design document we have as that would be more accurate than the details in feature request regarding how we will create and use external plugins. Here's the [link](https://github.com/kubernetes-sigs/kubebuilder/blob/master/designs/extensible-cli-and-scaffolding-plugins-phase-2.md) to the phase 2 external plugins design document.